remote-api/src/main/java/org/alfresco/rest/api/impl/CategoriesImpl.java

```diff
@@ -111,6 +112,11 @@ public Category getCategoryById(final StoreRef storeRef, final String id, final
             category.setCount(categoriesCount.getOrDefault(category.getId(), 0));
         }
 
+        if (parameters.getInclude().contains(INCLUDE_PATH_PARAM))
```

Comment:
I don't know if `parameters.getInclude()` can be null here (when no query params are passed to the endpoint) but if can - then NPE is quite possible here.

Reply:
As far as I checked if params are null there is a empty List created so NPE shouldn't happen